examples/simple-nginx.md

```diff
@@ -54,16 +54,16 @@ Once the pods are created, you can list them to see what is up and running:
 kubectl get pods
 ```
 
-You can also see the replication controller that was created:
+You can also see the deployment that was created:
 
 ```bash
-kubectl get rc
+kubectl get deployment
 ```
 
-To delete the two replicated containers, delete the replication controller:
+To delete the two replicated containers, delete the deployment:
 
 ```bash
-kubectl delete rc my-nginx
+kubectl delete deployment my-nginx
```

Comment:
Suggest moving this to the bottom, otherwise the below `kubectl expose` would fail.